### PR TITLE
vfio: backport fixes

### PR DESCRIPTION
**Pull request by devimc**

### CODE REVIEW

**Review by GabyCT — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,10 +33,10 @@ We provide several tests to ensure Kata-Containers run on different scenarios
 and with different container managers.
 
 1. Integration tests to ensure compatibility with:
-   - [Kubernetes](https://github.com/kata-containers/tests/tree/2.0-dev/integration/kubernetes)
-   - [CRI-O](https://github.com/kata-containers/tests/tree/2.0-dev/integration/cri-o)
-   - [Containerd](https://github.com/kata-containers/tests/tree/2.0-dev/integration/containerd)
-2. [Stability tests](https://github.com/kata-containers/tests/tree/2.0-dev/integration/stability)
+   - [Kubernetes](https://github.com/kata-containers/tests/tree/main/integration/kubernetes)
+   - [CRI-O](https://github.com/kata-containers/tests/tree/main/integration/cri-o)
```
Comment (fidencio) [inline README.md:37]: This is currently gone.  I wonder if we should point at `stable-2.0` instead of `main` for all of those?

Comment (devimc) [reply README.md:37]: I don't know I just back ported the patch, what should we do?

Comment (fidencio) [reply README.md:37]: In the backported patch, refer to the `stable-2.0` branches instead of the `main` one.

Comment (devimc) [reply README.md]: done, thanks @fidencio

**Comment by fidencio** [comment]
@devimc, I'd say just drop `ci: rename 2.0-dev to stable-2.0` and manually remove the broken link to the CRI-O integration tests.  I guess this will be easier.

**Comment by fidencio** [comment]
@devimc, mind to rebase this one so we can get it merged?

**Comment by fidencio** [comment]
> @devimc, mind to rebase this one so we can get it merged?

Oh, seems that I was able to resolve the conflicts via the webui.  The world is getting better amd better. :-)

**Pull request merged**